src/frontend/src/pages/HomePage/components/GuestOrganizationInfo.tsx

```diff
@@ -0,0 +1,76 @@
+import { Card, Typography, useTheme } from '@mui/material';
+import { Grid } from '@mui/material';
+import { useCurrentOrganization } from '../../../hooks/organizations.hooks';
+import React from 'react';
+import { NERButton } from '../../../components/NERButton';
+import InfoRoundedIcon from '@mui/icons-material/InfoRounded';
+import { useAllLinkTypes, useAllUsefulLinks } from '../../../hooks/projects.hooks';
+import LoadingIndicator from '../../../components/LoadingIndicator';
+import ErrorPage from '../../ErrorPage';
+import { Stack } from '@mui/system';
+
+interface GuestOrganizationInfoButtonProps {
+  href?: string;
+  buttonText: string;
+  iconName: string;
+}
+
+const NERGuestButton: React.FC<GuestOrganizationInfoButtonProps> = ({ href, buttonText }) => {
+  return (
+    <Grid item xs={4}>
+      <NERButton variant="contained" fullWidth={true} style={{ justifyContent: 'flex-start' }} href={href}>
+        <InfoRoundedIcon style={{ color: 'white' }} />
+        <Typography color="white" noWrap={true} sx={{ marginLeft: 1 }}>
+          {buttonText}
+        </Typography>
+      </NERButton>
+    </Grid>
+  );
+};
+
+const GuestOrganizationInfo = () => {
+  const theme = useTheme();
+  const { data: organization, isLoading, isError, error } = useCurrentOrganization();
+  const {
+    data: usefulLinks,
+    isLoading: usefulLinksIsLoading,
+    isError: usefulLinksIsError,
+    error: usefulLinksError
+  } = useAllUsefulLinks();
+  const { data: linkTypes, isLoading: linkTypesIsLoading } = useAllLinkTypes();
+
+  if (isLoading) return <LoadingIndicator />;
+  if (isError) return <ErrorPage message={error?.message} />;
+
+  if (!usefulLinks || usefulLinksIsLoading || !linkTypes || linkTypesIsLoading) return <LoadingIndicator />;
+  if (usefulLinksIsError) return <ErrorPage message={usefulLinksError.message} />;
+
+  return (
+    <Card
+      sx={{
+        padding: 2,
+        bgcolor: theme.palette.background.paper,
+        borderRadius: '1rem',
+        marginTop: 5
+      }}
+      variant="outlined"
+    >
+      <Stack spacing={2}>
+        <Typography variant="h4">FinishLine by {organization?.name}</Typography>
```

Comment:
remove "Finishline by" in the title, although this was in the mock, we just want the organization name